package.json

```diff
@@ -16,7 +16,7 @@
     "@bcrs-shared-components/bread-crumb": "1.0.2",
     "@bcrs-shared-components/business-lookup": "1.0.2",
     "@bcrs-shared-components/certify": "2.0.0",
-    "@bcrs-shared-components/completing-party": "2.0.0",
+    "@bcrs-shared-components/completing-party": "2.0.5",
```

Comment:
You will need to change this to 2.0.6 once your shared components PR is merged and the component is published.

Reply:
Did you look over the changes from v2.0.0 to v2.0.5 to make sure this project is still compatible with them?

Reply:
Yes. `npm i` succeeded.

Reply:
Fine, but did you look over the changes to the shared component since it was last used in this project? I see it went from v2.0.0 to v2.0.5, so some things were updated. Do those changes affect this UI?

Reply:
The latest CP component before my updates was 2.0.3 in the shared project, but Create UI was using 2.0.0. My updates will be in 2.0.5 (current version) and 2.0.6 (after the next merge). This UI doesn't use the CompletingParty from the shared component project, so changing the CP in the shared component won't affect this UI. We have to keep an eye for the Edit UI though.

Reply:
I'm confused. Create UI was using 2.0.0 but this UI doesn't use the shared CompletingParty? Are you saying it should not have been imported previously?

Reply:
I found this: https://github.com/bcgov/business-create-ui/blob/00f24266b9387dd9d92a6de709e5238f1befb153/src/views/DissolutionFirm/DissolutionFirm.vue#L221

Reply:
Don't merge this PR without testing this ^^

Reply:
Okay. I'll make the test case.

Reply:
I'd be content with you looking at what changed between before-version and now-version and then checking that the code that uses it is still OK :)